app/assets/javascripts/services/miq_service.js

```diff
@@ -53,6 +54,14 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', funct
     $('#flash_msg_div').text("");
   };
 
+  this.miqFlashLater = function(msgObj) {
```

Comment:
I understand, right so this makes sense. I had the impression that declaring a variable as global as it e.g. in `app/assets/javascripts/services/post_service.js` : 
```
/* global miqFlashLater */
```
is the way to deal with this problem (make CC and similar tools happy)

Yes, but from clean code perspective, that you did is surely better. I'd just love to have it done the same way everywhere (so probably `postservice.js`) needs fixing.